### PR TITLE
Fix config copy issue

### PR DESCRIPTION
**Pull request by shanmugapriya-tr**
### :nut_and_bolt: Description: What code changed, and why?
Introduced InputFilePrefix - to hold the formatted file name to be used everywhere

### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-3217

### :+1: Definition of Done
User should be able to patch config present in any folder path

### :athletic_shoe: How to Build and Test the Change
Checkout the branch
rebuild components/automate-cli

use the package to test changes

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
Screenshot 2023-06-05 at 8 55 56 PM
Screenshot 2023-06-05 at 8 55 33 PM
Screenshot 2023-06-05 at 8 55 15 PM
Screenshot 2023-06-05 at 8 53 46 PM

Screenshot 2023-06-05 at 3 34 50 PM
Screenshot 2023-06-05 at 3 36 18 PM

### CODE REVIEW

**Review by YashviJain01 — APPROVED** [review]

**Review by rishabhjhs — APPROVED** [review]

File: components/automate-cli/cmd/chef-automate/config.go [hunk 1]
```diff
@@ -357,7 +358,9 @@ func runPatchCommand(cmd *cobra.Command, args []string) error {
 		}
 
 		configFile := args[0]
-		frontendCmd := fmt.Sprintf(FRONTEND_COMMAND, PATCH, "frontend"+"_"+timestamp+"_"+configFile, dateFormat)
```
Comment (vivek-yadav) [inline components/automate-cli/cmd/chef-automate/config.go:360]: This line was the root cause of the issue, as the configFile = args[0] was appended to cmd, which later failed as relative path was different. when the config was located some other place.

**Review by vipin230 — APPROVED** [review]

File: components/automate-cli/cmd/chef-automate/cmdUtils.go [hunk 2]
```diff
@@ -31,6 +31,7 @@ type CmdInputs struct {
 	SkipPrintOutput          bool
 	HideSSHConnectionMessage bool
 	MutipleCmdWithArgs       map[string]string
+	InputFilesPrefix         string
```
Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/cmdUtils.go:34]: Move this line up, after `InputFiles               []string `
```suggestion
	InputFilesPrefix         string
```

Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/cmdUtils.go:34]: Move this line up, after the `InputFiles`

File: components/automate-cli/cmd/chef-automate/config.go [hunk 3]
```diff
@@ -395,10 +401,11 @@ func runPatchCommand(cmd *cobra.Command, args []string) error {
 				InputFiles:               []string{configFile},
 				ErrorCheckEnableInOutput: true,
 				NodeType:                 configCmdFlags.chef_server,
+				InputFilesPrefix:         chefserverPrefix,
 			},
 		}
-
-		postgresqlCmd := fmt.Sprintf(BACKEND_COMMAND, dateFormat, "postgresql", "%s", "postgresql"+"_"+timestamp+"_"+configFile)
+		postgresqlPrefix := "postgresql" + "_" + timestamp + "_"
```
Comment (ArvinthC3000) [inline components/automate-cli/cmd/chef-automate/config.go:407]: What happens if the user patches the config with the **same file name**?
If that is working properly, and we don't need to keep track of all the configs that are patched previously, I am good with this approach. Else consider adding a timestamp too.

File: components/automate-cli/cmd/chef-automate/cmdUtils.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (shanmugapriya-tr) [reply components/automate-cli/cmd/chef-automate/cmdUtils.go:34]: This is not the entire path. This is just the file name prefix

**Review by vivek-yadav — APPROVED** [review]

**Review by ArvinthC3000 — APPROVED** [review]

**Pull request merged**